src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -466,6 +466,7 @@ class GlobalSelectionHeader extends React.Component {
 
   getBackButton = () => {
     const {organization, location} = this.props;
+    const backLocation = {...location};
```

Comment:
Do you use this anywhere?

Reply:
No, sorry. A relic from an idea of just overriding the back button's link to not include the locked project (if they came from the issue list page). I'll remove it.